### PR TITLE
Send reminder of IFComp rules to authors

### PR DESCRIPTION
**Pull request by mjmusante**
After each game is registered, we send an email message to remind authors of the comp rules.

### CODE REVIEW

File: IFComp/lib/IFComp/Schema/Result/User.pm [hunk 1]
```diff
@@ -399,6 +399,15 @@ sub send_password_reset_email {
     $self->_send_email_and_reset_token('reset_password');
 }
 
+sub send_judge_reminder_email {
```
Comment (adherzog) [inline IFComp/lib/IFComp/Schema/Result/User.pm:402]: Can we change this method to `send_author_reminder_email`?

**Review by adherzog — CHANGES_REQUESTED** [review]
One change requested - otherwise looks great.

File: IFComp/lib/IFComp/Schema/Result/User.pm (hunk 1 above)
Comment (adherzog) [reply IFComp/lib/IFComp/Schema/Result/User.pm:402]: Oh, and tidy please.

Comment (mjmusante) [reply IFComp/lib/IFComp/Schema/Result/User.pm]: Thanks - not sure what I was thinking. I also need to tidy, but I'll do that after work tonight

**Pull request closed without merging**